### PR TITLE
Add MANTRA Chain TESTNET (mantrachain-testnet-1)

### PR DESCRIPTION
**Pull request by martinhalford**
# Request to add MANTRA Chain TESTNET to Keplr

Please review our request to add MANTRA Chain TESTNET to Keplr.

MANTRA is running a TESTNET in preparation for MAINNET launch in Q1 2024.

Details relating to our chain can be seen at: [https://mantrachain.io](https://mantrachain.io)

Out Litepaper can be seen at: [https://docs.mantrachain.io](https://docs.mantrachain.io)

Feel free to reach out to us on our [Discord](https://discord.gg/gfks4TwAJV) Server.

Any question please email: contact@mantrachain.io

Thank you for your consideration!

**The MANTRA Chain Team**

### CODE REVIEW

File: cosmos/mantrachain-testnet.json [hunk 1]
```diff
@@ -0,0 +1,50 @@
+{
+  "chainId": "mantrachain-testnet-1",
+  "chainName": "MANTRA Chain Testnet",
+  "chainSymbolImageUrl": "https://github.com/chainapsis/keplr-chain-registry/blob/main/images/mantrachain-testnet/chain.png",
```
Comment (HeesungB) [inline cosmos/mantrachain-testnet.json:4]: @martinhalford Hello.

Could you fix `chainSymbolImageUrl`?
to `https://raw.githubusercontent.com/chainapsis/keplr-chain-registry/main/images/mantrachain-testnet/chain.png`
from `https://github.com/chainapsis/keplr-chain-registry/blob/main/images/mantrachain-testnet/chain.png`

Comment (martinhalford) [reply cosmos/mantrachain-testnet.json]: @HeesungB - Update made and commit pushed.  Thanks!

**Comment by HeesungB** [comment]
@martinhalford Thank you for contribution!

**Comment by martinhalford** [comment]
Thank you @HeesungB!  Much appreciated!!

**Pull request merged**